src/models.py

```diff
@@ -59,13 +59,13 @@ def save(self, session, **kwargs):
 # Model StatesPerDay -> Date(Date), Country(String),
 # State(String), newCases(Int), totalCases(Int)
 class StatesPerDay(db.Model):
-    __tablename__ = 'STATES_PER_DAY'
+    __tablename__ = 'statesperday'
     id = db.Column(db.String(255), primary_key=True)
-    date = db.Column(db.DateTime, nullable=False)
-    country = db.Column(db.String(255), primary_key=True)
-    state = db.Column(db.String(255), nullable=False)
-    new_cases = db.Column(db.Integer)
-    total_cases = db.Column(db.Integer)
+    date = db.Column(db.DateTime)
```

Comment:
@jeduardo824 e @prschorn  será que a coluna date não deveria ser apenas do tipo date, sem o time? 
https://docs.sqlalchemy.org/en/13/core/type_basics.html
Ver: class sqlalchemy.types.Date

Reply:
Pode ser, acho que vai depender dos dados, se terão timestamp ou não.

Reply:
Boa observação:

CREATE TABLE public.statesperday (
id serial NOT NULL,
“date” date NULL,
country varchar NULL,
state varchar NULL,
newcases int4 NULL,
totalcases int4 NULL,
CONSTRAINT statesperday_pkey PRIMARY KEY (id)
);

a coluna "date" deve ser do tipo Date.

Reply:
Acho que só Date mesmo